setup.py

```diff
@@ -22,7 +18,8 @@
 
 here = pathlib.Path(__file__).parent
 
-if (here / '.git').exists() and not (here / 'vendor/http-parser/README.md').exists():
+if ((here / '.git').exists() and
```

Comment:
if you're refactoring this already, i'd do smth like 
```suggestion
IS_GIT_REPO = (here / '.git').exists()

if (IS_GIT_REPO and
```